src/house/services/auth-thunks.js

```diff
@@ -33,6 +33,14 @@ export const updateUserThunk = createAsyncThunk(
   }
 );
 
+// export const followThunk = createAsyncThunk(
```

Comment:
nit: remove all the `console.log` and commented code, thanks

Reply:
updated and will include in my next pull request

Reply:
addressed